_config.yml

```diff
@@ -927,19 +927,15 @@ vendors:
   # Example:
   # velocity: https://cdn.jsdelivr.net/npm/velocity-animate@1.2.1/velocity.min.js
   # velocity: https://cdnjs.cloudflare.com/ajax/libs/velocity/1.2.1/velocity.min.js
-  velocity:
-
-  # Internal version: 1.2.1
-  # See: http://VelocityJS.org
-  # Example:
   # velocity_ui: //cdn.jsdelivr.net/npm/velocity-animate@1.2.1/velocity.ui.min.js
   # velocity_ui: //cdnjs.cloudflare.com/ajax/libs/velocity/1.2.1/velocity.ui.min.js
+  velocity:
   velocity_ui:
 
   # Internal version: 0.7.9
   # See: https://faisalman.github.io/ua-parser-js
   # Example:
-  # ua_parser: //cdn.jsdelivr.net/npm/ua-parser-js@0.7.17/src/ua-parser.min.js
+  # ua_parser: //cdn.jsdelivr.net/npm/ua-parser-js@0.7.9/src/ua-parser.min.js
```

Comment:
The latest version is 0.7.19
![image](https://user-images.githubusercontent.com/15902347/50504296-9361ff00-0aa7-11e9-8755-5d4e7359e0a1.png)
Is it possible to use the latest version?

Reply:
I found this in `_config.yml`:
```
# Be aware that you should use the same version as internal ones to avoid potential problems.
```
So I'm not sure if we need to update the internal version, in /source/lib/ua-parser-js/dist ?

Reply:
Dude, just enought to use:

`//cdn.jsdelivr.net/npm/ua-parser-js@0/src/ua-parser.min.js` – will be opened **latest** major version of any plugin (e.g. 0.7.19). It's info from jsdelivr.com, try to check it.

![image](https://user-images.githubusercontent.com/16944225/50514432-770d9480-0a9e-11e9-92ea-b0251c9238e5.png)

Reply:
Thanks, I have updated all links from `cdn.jsdelivr.net` in this way. Does anything else need to be done?

Reply:
Recheck it. Must be `@x` version. See my previous comment carefully.

Reply:
It said `omit the version completely to get the latest one` in jsdelivr.com ... so do you think it's necessary to add a `@x` after the package name?

Reply:
Yes, i tested it. Some libs not work without `@x` version. And in addition: major versions can broke all work, instead of minor versions.

Reply:
Then, in summary, if the internal version is 0.7.9, the latest version is 0.7.19, which one would you prefer?
```
cdn.jsdelivr.net/npm/ua-parser-js/src/ua-parser.min.js
cdn.jsdelivr.net/npm/ua-parser-js@0/src/ua-parser.min.js
cdn.jsdelivr.net/npm/ua-parser-js@0.7.9/src/ua-parser.min.js
cdn.jsdelivr.net/npm/ua-parser-js@0.7.19/src/ua-parser.min.js
```

Reply:
> cdn.jsdelivr.net/npm/ua-parser-js@0/src/ua-parser.min.js

Reply:
OK, fixed